Gordon360/Services/HousingService.cs

```diff
@@ -338,8 +326,13 @@ public int EditApplication(string username, string sess_cde, int applicationID,
                     }
                     else
                     {
+                        ApartmentApplicantViewModel nonMatchingApplicant = new ApartmentApplicantViewModel
+                        {
+                            ApplicationID = existingApplicant.AprtAppID,
+                            Username = existingApplicant.Username, // Code for after we remade the AA_Applicants table
+                        };
                         // If the applicant is in the existing list but not in the new list of applicants, then we need to remove it from the database
-                        applicantsToRemove.Add(newMatchingApplicant);
+                        applicantsToRemove.Add(nonMatchingApplicant);
```

Comment:
This is actually a rather important bug, which was pointed out by @EjPlatzer a while ago